### PR TITLE
Fix problem with old/deprecated (?) gcc clobbering libstdcxx-ng on linux

### PR DESCRIPTION
**Pull request by ceball**
This seems to work locally for me, allowing filigree to work. Will try to figure out a better way to make this happen, so probably don't merge yet. As far as I know, nobody's being held up by filigree not working on linux (at least in this env).

### CODE REVIEW

**Comment by ceball** [comment]
Actually, maybe we should merge this to fix the website (filigree hasn't worked at https://pyviz.github.io/EarthSim/user_guide/Specifying_Meshes.html, but I think it would if this PR were merged)?

File: .travis.yml [hunk 1]
```diff
@@ -29,6 +29,12 @@ jobs:
         - conda env create
         - source activate earthsim
         - doit develop_install
+        ###
+        # TODO: figure out what's depending on (deprecated?) gcc package; see https://github.com/pyviz/EarthSim/issues/97
+        - pushd /home/travis/miniconda/envs/earthsim/lib/ && rm -rf libstdc++.so.6 && ln -s libstdc++.so.6.0.24 libstdc++.so.6 && popd
```
Comment (philippjfr) [inline .travis.yml]: Thanks for figuring that out.

**Comment by philippjfr** [comment]
I'll merge once tests pass.

**Comment by ceball** [comment]
Yes, makes sense to wait for the tests in case my little refactor of the hack has a typo. You won't know it's fixed the website until after merging, though (but it worked for me locally).

**Pull request merged**